src/main/java/domain/LottoTicket.java

```diff
@@ -0,0 +1,61 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoTicket {
+	private static final int LOTTO_NUMBER_COUNT = 6;
+	public static final int PRICE = 1000;
+
+	private final List<LottoNumber> lottoNumbers;
+
+	private LottoTicket(List<LottoNumber> lottoNumbers) {
+		validateSize(lottoNumbers);
+		validateDuplicate(lottoNumbers);
+		this.lottoNumbers = lottoNumbers;
+	}
+
+	public static LottoTicket of(List<Integer> lottoNumbers) {
+		return new LottoTicket(lottoNumbers.stream().map(LottoNumber::new).collect(Collectors.toList()));
```

Comment:
한 줄에 하나의 함수만 호출하면 가독성이 조금 더 좋아질 것 같아요. 😃
또한 이렇게 적절히 변수로 추출하는 것 또한 가독성에 도움을 줄 것 같네요. 😉

```suggestion
            List<LottoNumber> lottoNumbers = lottoNumbers.stream()
                .map(LottoNumber::new)
                .collect(Collectors.toList())
            return new LottoTicket(lottoNumbers);
```

Reply:
반영했습니다.
https://github.com/next-step/java-lotto-kakao/pull/14/commits/27d91a33da6295dcfcf0dcbae3695e7bc58394ad

Reply:
반영했습니다.
https://github.com/next-step/java-lotto-kakao/pull/14/commits/27d91a33da6295dcfcf0dcbae3695e7bc58394ad